### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by leonhartX**
Signed-off-by: Ke Xu <leonhartx.k@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @ryankeairns, @cwebberOps, @davidwrede, @smith and @juliandunn to be potential reviewers

File: www/source/docs/continuous-deployment-overview.html.md [hunk 1]
```diff
@@ -48,3 +48,9 @@ Each of these phases runs per environment (acceptance, union, rehearsal, deliver
 ### Supervisor Configuration
 
 In order to enable automatic deployment in each environment during the provision stage, you should [configure the supervisor](/docs/run-packages-update-strategy/) to use a named view as its depot URL, and set its update strategy. By convention, it's best to name the depot view the same as the service group name.
+
+<hr>
+<ul class="main-content--link-nav">
+  <li>Continue to the next topic</li>
+  <li><a href="/docs/nternals-overview">Habitat internals</a></li>
```
Comment (davidwrede) [inline www/source/docs/continuous-deployment-overview.html.md]: Looks like you're missing an "i" in /docs/nternals-overview.

**Comment by davidwrede** [comment]
@leonhartX, thanks so much for fixing the broken links and adding in the transition links! I left a comment on one transition link, but other than that, this looks great.

**Comment by leonhartX** [comment]
@davidwrede thanks for review, have fixed that typo.

**Comment by smith** [comment]
@thesentinels r+

gif-keyboard-5836740274201093045

**Comment by thesentinels** [comment]
:pushpin: Commit d9e098f has been approved by `smith`

**Comment by thesentinels** [comment]
:hourglass: Testing commit d9e098f with merge 89379e4...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.com/habitat-sh/habitat/builds/26916155)

**Pull request merged**